### PR TITLE
py file added

### PR DESCRIPTION
**Pull request by VahidHosseinzadeh**
I added the main.py which is a sqlalchemy code for creating tables for hep-analysis

### CODE REVIEW

**Review by javadebadi — CHANGES_REQUESTED** [review]
In the 1st line use the following style to import: [link](https://stackoverflow.com/questions/14376900/is-there-a-recommended-format-for-multi-line-imports)

In the 5th line:
Create a new file "settings.py" and add a new variable in that which contains information about connection string such as 
`DB_CONNECTION_STRING = "sqlite+pysqlite:///:memory:"`
Then import the DB_CONNECTION_STRING in this module.

Thank you for your code.
I will review more after you apply these changes.

File: setting.py [hunk 1]
```diff
@@ -0,0 +1 @@
+DB_CONNECTION_STRING = "sqlite+pysqlite:///:memory:"
```
Comment (javadebadi) [inline setting.py:1]: Nice!

File: main.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+engine = create_engine(DB_CONNECTION_STRING, echo=True)
+
+conn = engine.connect() 
+
+meta_HEP = MetaData()
+
+#Creatinf Tables using Core commands: 
+Job_table = Table(
+        "Jobs",
```
Comment (javadebadi) [inline main.py:30]: Please align the names of tables in Table() as the `Columns`.
For example, in `Job_table` definition the `"Jobs"` string is not aligned with Columns in the below

File: main.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    Column('region_id', ForeignKey('Country_table.region_id'), nullable=False),
+    Column('rank_id', ForeignKey('Rank_table.rank_id'), nullable=False),
+    )
+
+
+Field_table = Table(
+        "Fields",
+    meta_HEP,
+    Column('field_id', Integer, primary_key=True),
+    Column('field_title', String(30))
```
Comment (javadebadi) [inline main.py]: Usually, when we write the elements of a list of parameters of function in a column way, we should add a trailing comma at the end of the last item. For example, here you have `Column('field_title', String(30))` but in google style you need to write `Column('field_title', String(30)),`.

File: main.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+class JobFileds(Base):
+    __table__ = JobFileds_table
+
+    def __repr__(self):
+         return f"Fields({self.field_title!r})"
+
+
+
+
+if __name__ == '__mane__':
```
Comment (javadebadi) [inline main.py]: This must be changed o `if __name__ == '__main__'`

**Review by javadebadi — CHANGES_REQUESTED** [review]
Hi vahid,
Thank you for previous changes.

I added some more changes to the styles which you can see.

In addition, I have a question about using ORM.
Why do you use the ORM? we can create tables using SQLAlchemy Core without any need to use ORM. Is there any specific reason that you have defined The ORM? Is this because of future needs?

File: main.py (hunk 2 above)
Comment (VahidHosseinzadeh) [reply main.py:30]: Hi Javad, Yeah sure.

File: main.py (hunk 4 above)
Comment (VahidHosseinzadeh) [reply main.py]: You mean I should delete `Base.metadata.create_all(engine)`?

Comment (VahidHosseinzadeh) [reply main.py]: oh sorry, why did I write mane :D. yeah got it. why there is no error? it is strange ...

**Comment by VahidHosseinzadeh** [comment]
> Hi vahid, Thank you for previous changes.
> 
> I added some more changes to the styles which you can see.
> 
> In addition, I have a question about using ORM. Why do you use the ORM? we can create tables using SQLAlchemy Core without any need to use ORM. Is there any specific reason that you have defined The ORM? Is this because of future needs?

I went over the SQLAlchemy document in their site. As you said there are two method for implementing sql in sqlalchemy. 
Core and ORM. as far as I understand, there is no need for object relational mapping (ORM) method unless we want to create a database based on some classes and their objects as tables and instances (rows). We do not need this, right?
a question, I still read the document but let me ask this: now the code is creating the tables through constructing the classes or by the Table command of Core? or both maybe, I mean is it creating two copy of them?

BTW, there is a new version of sqlalchemy (2) on the way.

**Comment by javadebadi** [comment]
Hi Vahid,
Thank you.
In fact, we can use the ORM of SQLAlchemy.
I will merge your changes to the master branch.

**Pull request merged**